traceabilityViewer/traceabilityViewer/settings.py

```diff
@@ -0,0 +1,136 @@
+"""
+Django settings for traceabilityViewer project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+from neomodel import config
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-9@1+y@2v04x9t4r=ih5-%j7ueh(84j6($3tt6js&pd%ozl91du"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Based on that warning, this should become configurable as well.

Reply:
Resolved in ecb55b948db709b70506e6b9c1d0b6602465fa58

Reply:
Resolved in 8d6e5d4566ca65f4d6965480bd60c60ce5ce4a07

Reply:
Better, but you didn't make it configurable. Isn't it useful to turn debug on during development?

Reply:
Yes, I think the errors gained from it are also interesting. I will make it configurable in `.env` .

Reply:
Resolved in b9ca41e7d993cef9602617b2af1166553906128b